### ISSUE TITLE
Build rendered Certificate view

### ISSUE THREAD

**Issue by bedinotti** [blockchain-certificates/BlockcertsFramework-iOS#5]
This is the visually designed display that looks like a real certificate.

**Comment by bedinotti** [comment]
The placeholder view for this is complete:
<img width="319" alt="screen shot 2016-08-26 at 3 29 46 pm" src="https://cloud.githubusercontent.com/assets/547866/18021045/028ff7c6-6ba2-11e6-96b6-85e21f60ce2b.png">

However, I don't really want to consider this task done until that view is a bit more refined.

**Comment by bedinotti** [comment]
I created a [first draft](https://github.com/learningmachine/digital-certs-wiki/wiki/Certificate-Display-Guidelines) on community guidelines for displaying certificates. I need to:
- Get feedback on the proposed large & small designs
- Move that post to something in the blockcerts domain, so others can see it
- Re-render the images with a published certificate that others could use to test their rendering.

I also updated the display to look a bit more respectable. But not much.

**Comment by bedinotti** [comment]
I'm going to call this good enough to ship. Display has been updated so it's fully responsive, and renders consistently on different device sizes. I also added in the description.

Signatures have been removed. I'm going to wait to add those back in until we iterate the format to formally support titled signatures. And even then, it may not make it to the mobile display.

![screen shot 2016-09-23 at 3 40 04 pm](https://cloud.githubusercontent.com/assets/547866/18802122/38d5bb1e-81a4-11e6-86ae-6f722da05658.png)
![simulator screen shot sep 23 2016 3 40 48 pm](https://cloud.githubusercontent.com/assets/547866/18802121/38d53734-81a4-11e6-87ed-a180743502ac.png)
